code/modules/reagents/chemistry/machinery/chem_heater.dm

```diff
@@ -115,6 +124,10 @@
 		on = !on
 		. = 1
 
+	if(href_list["toggle_autoeject"])
+		auto_eject = !auto_eject
+		. = 1
```

Comment:
`TRUE` here. it's a boolean value so the `TRUE`/`FALSE` defines are prefered.

Reply:
Done, I also went and replaced the other 1/0's in the file with the defines

Reply:
Very nice. Thanks for the added refactor!